Header header
#The command values, to be transmitted
uint8[32] raw
#never transmitted
uint8 debugflags
float32[10] debugvals
int32 debugtype
